internal/builds/serializers.go

```diff
@@ -0,0 +1,57 @@
+package builds
```

Comment:
Can you add some file-level comments? I'm trying to add these to the files I have so it's easier to sort what type of logic should belong or shouldn't belong in a file.

Reply:
✅